doc/release-notes.md

```diff
@@ -4,36 +4,83 @@ release-notes at release time)
+- The testnet reactivation height for NU5 is set to **1,840,000**.
+
+Testnet nodes that upgrade to `zcashd v4.7.0` prior to block height `1,840,000` will follow
+the new testnet network upgrade. Testnet nodes that are running `zcashd v4.6.0-2` at that
+height will need to upgrade to `v4.7.0` and then run with `-reindex`.
+
 Mnemonic Recovery Phrases
 -------------------------
 
-The zcashd wallet has been modified to support BIP 39, which describes how to
-derive the wallet's HD seed from a mnemonic phrase.  The mnemonic phrase will
-be generated on load of the wallet, or the first time the wallet is unlocked,
-and is available via the `z_exportwallet` RPC call. All new addresses produced
-by the wallet are now derived from this seed using the HD wallet functionality
-described in ZIP 32 and ZIP 316. For users upgrading an existing Zcashd wallet,
-it is recommended that the wallet be backed up prior to upgrading to the 4.7.0
-Zcashd release.
-
-Following the upgrade to 4.7.0, Zcashd will require that the user confirm that
-they have backed up their new emergency recovery phrase, which may be obtained
-from the output of the `z_exportwallet` RPC call. This confirmation can be
-performed manually using the `zcashd-wallet-tool` utility that is supplied
-with this release (built or installed in the same directory as `zcashd`).
-The wallet will not allow the generation of new addresses until this
-confirmation has been performed. It is recommended that after this upgrade,
-that funds tied to preexisting addresses be migrated to newly generated
-addresses so that all wallet funds are recoverable using the emergency
-recovery phrase going forward. If you choose not to migrate funds in this
-fashion, you will continue to need to securely back up the entire `wallet.dat`
-file to ensure that you do not lose access to existing funds; EXISTING FUNDS
-WILL NOT BE RECOVERABLE USING THE EMERGENCY RECOVERY PHRASE UNLESS THEY HAVE
-BEEN MOVED TO A NEWLY GENERATED ADDRESS FOLLOWING THE 4.7.0 UPGRADE.
+The zcashd wallet has been modified to support BIP 39, which describes how to derive the
+wallet's HD seed from a mnemonic phrase.  The mnemonic phrase will be generated on load of
+the wallet, or the first time the wallet is unlocked, and is available via the
+`z_exportwallet` RPC call. All new addresses produced by the wallet are now derived from
+this seed using the HD wallet functionality described in ZIP 32 and ZIP 316. For users
+upgrading an existing Zcashd wallet, it is recommended that the wallet be backed up prior
+to upgrading to the 4.7.0 Zcashd release.
+
+Following the upgrade to 4.7.0, Zcashd will require that the user confirm that they have
+backed up their new emergency recovery phrase, which may be obtained from the output of
+the `z_exportwallet` RPC call. This confirmation can be performed manually using the
+`zcashd-wallet-tool` utility that is supplied with this release (built or installed in the
```

Comment:
Is there any separate doc for the wallet tool or are we relying on the help text?

Reply:
Just the help text at present.

Reply:
Also note that `zcashd-wallet-tool` doesn't require arguments if you're using the default settings for zcashd's datadir path, port, and RPC authentication. If you've changed those and run it with no arguments, it will tell you what is wrong.